hawc/apps/animal/actions/model_metadata.py

```diff
@@ -0,0 +1,89 @@
+from pydantic import BaseModel
+from rest_framework import serializers
+
+from hawc.apps.animal import models
+from hawc.apps.assessment.models import DoseUnits, Species, Strain
+from hawc.apps.common.actions import BaseApiAction
+
+
+def tuple_to_dict(tuple):
+    return {e[0]: e[1] for e in tuple}
+
+
+class SpeciesSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Species
+        fields = ("id", "name")
+
+
+class StrainSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Strain
+        fields = ("id", "species_id", "name")
+
+
+class DoseUnitsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = DoseUnits
+        fields = ("id", "name")
+
+
+class NoInput(BaseModel):
+    pass
+
+
+class AnimalMetadata(BaseApiAction):
+    """
+    Generate a dictionary of field choices for all animal models.
+    """
+
+    input_model = NoInput
+
+    def experiment_metadata(self):
+        metadata = {}
+        metadata["type"] = tuple_to_dict(models.Experiment.EXPERIMENT_TYPE_CHOICES)
+        return metadata
+
+    def animal_group_metadata(self):
+        metadata = {}
+        species = Species.objects.all()
+        strains = Strain.objects.all()
+        metadata["sex"] = models.AnimalGroup.SEX_DICT
+        metadata["generation"] = models.AnimalGroup.GENERATION_DICT
+        metadata["species"] = SpeciesSerializer(species, many=True).data
+        metadata["strains"] = StrainSerializer(strains, many=True).data
+        metadata["lifestage"] = tuple_to_dict(models.AnimalGroup.LIFESTAGE_CHOICES)
+        return metadata
+
+    def dosing_regime_metadata(self):
+        metadata = {}
+        metadata["route_of_exposure"] = models.DosingRegime.ROUTE_EXPOSURE_CHOICES_DICT
+        metadata["positive_control"] = tuple_to_dict(models.DosingRegime.POSITIVE_CONTROL_CHOICES)
+        metadata["negative_control"] = tuple_to_dict(models.DosingRegime.NEGATIVE_CONTROL_CHOICES)
+        return metadata
+
+    def dose_group_metadata(self):
+        metadata = {}
+        dose_units = DoseUnits.objects.all()
```

Comment:
instead of a query and a serializer, I think we can just use [values](https://docs.djangoproject.com/en/3.1/ref/models/querysets/#values)  with a queryset, for this and all the other serializer cases I think

```python
list(DoseUnits.objects.all().values('id', 'name'))
```